onnxscript/tools/transformers_models/llama.py

```diff
@@ -92,3 +92,75 @@ def generate_example_inputs(batch: int, seq: int, vocab_size: int):
         example_args_collection.append(generate_example_inputs(b, s, vocab_size))
 
     return LlamaModelWrapper(config), example_args_collection, dynamic_shapes
+
+
+def get_llama_model_config(
+    warmup: int = 5,
+    repeat: int = 10,
+    config: str = "small",
+    num_hidden_layers: int = 1,
+    implementation: str = "eager",
+    dynamic_shapes: bool = False,
+    with_mask: bool = True,
+) -> tuple[Any, list[tuple[torch.Tensor, ...]], dict]:
+    """
+    Returns a model Phi to test or benchmark.
+
+    Args:
+        warmup: number of inputs to generate
```

Comment:
no big deal but for future reference: Capitalize the start of the phase and end with a period. See https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html (note we don't need to include type information as the example did because we have type info in the function signature already).

`warmup: Number of inputs to generate.`

Reply:
done